css/src/components/layout.scss

```
@@ -187,3 +186,24 @@ $three-quarters-widescreen: math.div($breakpoint-widescreen, 4) * 3;
 		}
 	}
 }
+
+.layout.layout-twin {
+	.layout-body-menu {
+		display: none;
+	}
+
+	.layout-body {
+		grid-template: auto auto auto 1fr auto / minmax(0, 1fr);
+		grid-template-areas: 'header' 'hero' 'main' 'aside' 'footer';
+
+		// note that to make some extra room this layout is not constrained by the widescreen breakpoint like others
+		@include tablet {
+			grid-template: auto auto 1fr auto / minmax(0, 1fr) minmax(0, 1fr);
+			grid-template-areas:
+				'header header'
+				'hero hero'
+				'main aside '
```

suuuuper nit, but should this be

```suggestion
				'main aside'
```